views/faucet.pug

```diff
@@ -78,19 +78,9 @@ block content
       }
     }
 
-    async function enableEthereum() {
-      if (window.ethereum) {
-        metamask = new Web3(window.ethereum);
-        try {
-          await window.ethereum.enable();
-        } catch (error) {
-          console.log(error);
-        }
-      }
-    }
-
     window.addEventListener('load', async () => {
       await enableEthereum();
+      validateChainId(!{JSON.stringify(data.chainId)});
```

Comment:
What is the `!{` part doing?

Reply:
`!{` is for unescaped strings, in this case the rendered js will look like:
```js
alidateChainId("66");
```
using `#{` would generate this invalid js code:
```js
validateChainId(&quot;66&quot;)
```
```